### ISSUE TITLE
Remove text style "Group title"

### ISSUE THREAD

**Issue by itsgrafik** [sonburn/symbol-organizer#143]
It would be good if "Group title" text style wouldn't be created and added to the library after running the script. This adds unneccessary style, and also it gets listed as a new style over and over if uploading to Zeplin per example. Thanks.

**Comment by stdk00** [comment]
yeah this is pretty annoying. I would like to apply any text-style to that group title.
